improvements-proposal.md

```diff
@@ -0,0 +1,9 @@
+## Questions
+
+1 - Right now the data for the posts is coming from a json file. What changes would you have to make to the application if it came from an API? In what type of hook should you use to fetch the data and why? What other considerations would you have to make?
+
+A: I would call the API using an async function inside a useEffect hook to allow the site to run smoothly while waiting on the data. The data would also go into either the state or the reducer if using redux.
```

Comment:
- How could you improve your current solution if the number of blog posts was really large (for example 10,000)?

- What are additional features you would consider adding now that the data is being requested from an API?

Reply:
Are the 10000 blog posts still being loaded from a json file?